src/NLog/Common/ConversionHelpers.cs

```diff
@@ -44,24 +44,39 @@ public static class ConversionHelpers
         /// <summary>
         /// Converts input string value into <see cref="System.Enum"/>. Parsing is case-insensitive.
         /// </summary>
-        /// <param name="inputValue">Input value</param>
-        /// <param name="resultValue">Output value</param>
+        /// <param name="value">Input value</param>
```

Comment:
Not that I don't like the names, but changing parameter on public-methods are breaking change.

Reply:
you're right, thanks